pubber/build.gradle

```diff
@@ -55,6 +55,7 @@ jacocoTestReport {
 checkstyle {
     ignoreFailures = false
     maxWarnings = 0
+    configFile = file('../etc/checkstyle.xml')
```

Comment:
There's already pubber/config/checkstyle/checkstyle.xml -- do you know how this configFile option plays with that?  Overall, I like the idea of a centralized checkstyle.xml, I just want to make sure that this isn't taking an inadvertent step backwards (i.e., ignoring the existing one).